flexinput/src/main/java/com/lytefast/flexinput/adapters/PhotoCursorAdapter.kt

```diff
@@ -64,6 +85,11 @@ class PhotoCursorAdapter(private val contentResolver: ContentResolver,
     super.onBindViewHolder(holder, position, payloads)
   }
 
+  override fun onViewRecycled(holder: ViewHolder) {
+    holder.loadingThumbnailJob?.cancel()
```

Comment:
please make this a method on the view holder itself and make the `loadingThumbnailJob` private within the view holder.